src/Compilers/Core/CodeAnalysisTest/AnalyzerAssemblyLoaderTests.cs

```diff
@@ -495,6 +498,13 @@ private static void VerifyDependencyAssemblies(AnalyzerAssemblyLoader loader, in
                 .GetAssemblies()
                 .Where(x => isInLoadFromContext(loader, x));
 
+            // When debugging the debugger will load this DLL and that can throw off the debugging 
+            // session so excude it here.
+            if (Debugger.IsAttached)
+            {
+                loadedAssemblies = loadedAssemblies.Where(x => !(x.GetName().Name == "Microsoft.VisualStudio.Debugger.Runtime.Desktop"));
```

Comment:
I'm not crazy, right? This is equivalent?
```suggestion
                loadedAssemblies = loadedAssemblies.Where(x => x.GetName().Name != "Microsoft.VisualStudio.Debugger.Runtime.Desktop");
```

Reply:
you're not not crazy.

Reply:
Think this is a left over from where I was comparing several names and had a grouping.